hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/operation/InitExecutionOperation.java

```diff
@@ -53,7 +53,6 @@ public InitExecutionOperation(long jobId, long executionId, int coordinatorMembe
         this.executionId = executionId;
         this.coordinatorMemberListVersion = coordinatorMemberListVersion;
         this.participants = participants;
-        this.executionId = executionId;
```

Comment:
We can also make the variable `final`

Reply:
It's deserialized in `readInternal()` so these fields can't be made final